lib/inferno/dsl/fhir_evaluator/config.rb

```diff
@@ -0,0 +1,53 @@
+module Inferno
+  module DSL
+    module FHIREvaluation
+      class Config
+        DEFAULT_FILE = File.join(__dir__, 'config', 'default.yml')
+        attr_accessor :data
+
+        # To-do: add config_file as arguments
+        def initialize(config_file = nil)
+          @data = if config_file.nil?
+                    YAML.load_file(File.absolute_path(DEFAULT_FILE))
+                  else
+                    YAML.load_file(File.absolute_path(config_file))
+                  end
+
+          raise(TypeError, 'Malformed configuration') unless @data.is_a?(Hash)
+
+          # def method_missing(name, *args, &)
```

Comment:
I assume the point of all this `method_missing` stuff and Section class was so that nested fields could be referenced as `config.rule.rule_name.whatever` instead of `config['rule']['rule_name']['whatever']`. If we don't want to do that anymore we should delete this rather than comment it out. Or if we do want to support that, uncomment this. I feel like it's nice but not necessary